backport https://github.com/blueyed/awesome/commit/cd94a1368786d9061b0f11e7a8d757b976a2697b.patch
Include tests/examples/CMakeLists.txt always
see https://bugs.gentoo.org/962597
fix unavailable check-examples target without doc enabled
--- a/awesomeConfig.cmake
+++ b/awesomeConfig.cmake
@@ -275,15 +275,15 @@ if(GENERATE_DOC)
     # Load the common documentation
     include(docs/load_ldoc.cmake)
 
-    # Use `include`, rather than `add_subdirectory`, to keep the variables
-    # The file is a valid CMakeLists.txt and can be executed directly if only
-    # the image artefacts are needed.
-    include(tests/examples/CMakeLists.txt)
-
     # Generate the widget lists
     include(docs/widget_lists.cmake)
 endif()
 
+# Use `include`, rather than `add_subdirectory`, to keep the variables
+# The file is a valid CMakeLists.txt and can be executed directly if only
+# the image artefacts are needed.
+include(tests/examples/CMakeLists.txt)
+
 # {{{ Configure files
 file(GLOB awesome_base_c_configure_files RELATIVE ${SOURCE_DIR}
     ${SOURCE_DIR}/*.c
